R/shinyapp.R

```diff
@@ -373,9 +376,6 @@ loadSupport <- function(appDir=NULL, renv=new.env(parent=globalenv()), globalren
   }
 
 
-  helpersDir <- file.path(appDir, "R")
-
-  disabled <- list.files(helpersDir, pattern="^_disable_autoload\\.r$", recursive=FALSE, ignore.case=TRUE)
   if (length(disabled) > 0){
     return(invisible(renv))
   }
```

Comment:
@krlmlr maybe simpler to move this early exit on disabled to L356 instead of extending the if clause?

Reply:
In the end, I think the warning was happening too early. By moving it down we can instead warn just before evaluating the code in the `R/` support directory.